src/olympia/lib/settings_base.py

```diff
@@ -83,7 +83,6 @@ def path(*folders):
 PNGCRUSH_BIN = 'pngcrush'
 
 FLIGTAR = 'amo-admins+fligtar-rip@mozilla.org'
```

Comment:
as much as I like my old jokes, we should probably change this email address (and constant) at some point.